### PR TITLE
Add omaha/smart_proxy_omaha debian build

### PR DESCRIPTION
**Pull request by lhw**
This plugin already exists in the RPM repos, supplied by @timogoebel. It can be build for:

* [x] Nightly
* [x] 1.20
* [x] 1.19
* [x] 1.18

I am not to sure about compat and debian versions as the ones used here are fairly old but that might be because of the internal build system.

### CODE REVIEW

File: plugins/ruby-foreman-omaha/debian/copyright [hunk 1]
```diff
@@ -0,0 +1,28 @@
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: ruby-foreman-omaha
+Source: https://github.com/theforeman/foreman_omaha
+
+Files: *
+Copyright: 2017-2018 Timo Göbel
```
[the remark this answers is not in the document]
Comment (lhw) [reply plugins/ruby-foreman-omaha/debian/copyright]: My bad. Just seemed to obvious to me. I fixed it. If you want an E-Mail there as well just let me know which one.

[the remark this answers is not in the document]
Comment (timogoebel) [reply plugins/ruby-foreman-omaha/debian/copyright]: I'm good. Thanks. :-)

**Comment by ekohl** [comment]
ok to test

**Comment by lhw** [comment]
@ekohl is the clone URL hardcoded for the tests? Seems like its trying to get lhw/foreman-packaging instead of HanseMerkur/foreman-packaging for the tests

**Comment by ekohl** [comment]
It does look like we assume the author is the same. Doesn't look like it's a trivial change in our current scripts, though a good one to make.

**Comment by lhw** [comment]
Alright. To not make this more complicated then it has to be i'll close this one and make a new PR coming from my account.

**Pull request closed without merging**